cluster/compose/localnet/conf/splice/sv/app.conf

```diff
@@ -37,6 +37,7 @@ canton {
   }
 
   scan-apps.scan-app {
+    latest-packages-only = true
```

Comment:
Let's add an env var to localnet to disable this. This is a breaking change in the end, while it makes sense to have it on by default I think, we need to:
1. Provide an easy way to go back to the prior state through an env var.
2. Mention this in the release notes as a breaking change and with the env var to go back.

This matters in particular for users that have their own Daml models that depend on splice-amulet. They might have compiled against an older version in which case this is not gonna work.